pkg/v1workermeta/api.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1workermeta
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/pingcap/dm/dm/config"
+	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/terror"
+	"github.com/pingcap/dm/pkg/utils"
+)
+
+// `var` rather than `const` for testing.
+var (
+	// metaPath is the meta data path in v1.0.x.
+	metaPath = "./dm_worker_meta"
```

Comment:
user may set another value of `meta-dir` in v1.0.x, so we may change this set by configuration.

Reply:
YES, users may set `meta-dir`. but I want to only support upgrading from DM-Ansible deployed v1.0.x to v2.0.x, and if deployed with DM-Ansible, this item should not be set.

BTW, I don’t really want to add new configuration items in v2.0.x 😭 .

How about checking if the path is a valid meta directory and only do something if it is?

Reply:
Okay, I think it's fine if DM-Ansible call this upgrading function, and we could warn user in manual upgrading doc.